Tests/AutoCollection/Statsbeat.tests.ts

```diff
@@ -245,13 +246,14 @@ describe("AutoCollection/Statsbeat", () => {
             setTimeout(() => {
                 let envelope = spy.args[0][0][1];
                 let baseData: Contracts.MetricData = envelope.data.baseData;
-                assert.equal(baseData.metrics[0].name, "Instrumentation");
+                assert.equal(baseData.metrics[0].name, "Feature");
```

Comment:
all the properties in the custom dimensions are using lower case. please be consistent.

Reply:
This is the metric name, must be fist letter upper case, please confirm

Reply:
oops, sorry miss read that.  correct.